### PR TITLE
fix for the CVE-2022-31836 4ca2780dbf19d137746041886525fdebe594e50a

### PR DESCRIPTION
**Pull request by Nandavardhan8**

### CODE REVIEW

**Comment by flycash** [comment]
Hi, can you resolve the conflicts? sorry for late reply. You don't need to update the CHANGELOG because I prefer to force the users to read the release log.

It's much easier to maintain.

**Review by flycash — APPROVED** [review]

**Comment by gnanakeethan** [comment]
@flycash it seems the changes are empty. Is this correct?

File: server/web/session/ledis/http:/host:port/LOCK
Comment (chumaumenze) [inline server/web/session/ledis/http:/host:port/LOCK:1]: @flycash this file does not have a valid path and will cause issues on Windows and go mod resolution.

```
go: github.com/p/n/models imports
	github.com/beego/beego/v2/client/orm: create zip: server/web/session/ledis/http:/host:port/LOCK: malformed file path "server/web/session/ledis/http:/host:port/LOCK": invalid char ':'
server/web/session/ledis/http:/host:port/goleveldb_data/000001.log: malformed file path "server/web/session/ledis/http:/host:port/goleveldb_data/000001.log": invalid char ':'
server/web/session/ledis/http:/host:port/goleveldb_data/CURRENT: malformed file path "server/web/session/ledis/http:/host:port/goleveldb_data/CURRENT": invalid char ':'
server/web/session/ledis/http:/host:port/goleveldb_data/LOCK: malformed file path "server/web/session/ledis/http:/host:port/goleveldb_data/LOCK": invalid char ':'
server/web/session/ledis/http:/host:port/goleveldb_data/LOG: malformed file path "server/web/session/ledis/http:/host:port/goleveldb_data/LOG": invalid char ':'
server/web/session/ledis/http:/host:port/goleveldb_data/MANIFEST-000000: malformed file path "server/web/session/ledis/http:/host:port/goleveldb_data/MANIFEST-000000": invalid char ':'
```

**Comment by ExplosiveBattery** [comment]
could anyone help this?

**Pull request merged**